lms/djangoapps/discussion/rest_api/tests/test_api.py

```diff
@@ -1741,8 +1741,19 @@ def test_basic_in_blackout_period_with_user_access(self, mock_emit):
             "comment_list_url": "http://testserver/api/discussion/v1/comments/?thread_id=test_id",
             "read": True,
             "editable_fields": [
-                "abuse_flagged", "anonymous", "closed", "following", "pinned",
-                "raw_body", "read", "title", "topic_id", "type", "voted"
+                "abuse_flagged",
+                "anonymous",
+                "close_reason_code",
+                "closed",
+                "edit_reason_code",
+                "following",
+                "pinned",
+                "raw_body",
+                "read",
+                "title",
+                "topic_id",
+                "type",
+                "voted",
```

Comment:
I've reflown those test case lists because it makes it easier to add new items when necessary, while keeping them sorted, as the tests require them to be in lexicographic ordering (they're serialized from sets, which are unordered).

Reply:
Sounds good! Thanks for doing this.